chore: link session id to backend (#6250)
PostHog/posthog.com: contents/docs/session-replay/troubleshooting.mdx

Having trouble with recordings? Below are some tips for getting past some common issues

The most common solution is...

#### Update posthog-js

> We're always making improvements to the recordings feature, so you'll want to make sure that you're running the latest version of posthog-js on your website.

Reviewer: ```suggestion
We're always making improvements to the recordings feature, so you'll want to make sure that you're running the latest version of `posthog-js` on your website.
```

> To check the version that you're using, you can run the following in your console:

Reviewer: ```suggestion
To check the version that you're using, you can run the following in your browser console:
```
I'm not sure this is right, so if it can be run in other consoles, feel free to ignore.

If you see errors like this, please add `app.posthog.com` to the list of permitted domains in your webservers CORS configuration.

## Filtering can't find my custom events

> The most common reason for this is that you're trying to filter by events that weren't sent by the Browser SDK. For example from another tool like Segment or from one of our back-end SDKs.

Reviewer: ```suggestion
The most common reason for this is that you're trying to filter by events that weren't sent by the web SDK. For example, from another tool like Segment or one of our backend SDKs.
```

> Session replay relies on the `$session_id` property added to events by the browser SDK to link events to sessions.

Author: ```suggestion
Session replay relies on the `$session_id` property added to events by the Browser SDK to link events to sessions.
```

> Session replay relies on the `$session_id` property added to events by the Browser SDK to link events to sessions.

Reviewer: ```suggestion
Session replay relies on the `$session_id` property added to events by the web SDK to link events to sessions.
```

> To workaround this you can use the `posthog.onSessionId` method in the Browser SDK to register a callback that will be called each time the session id changes. You can send this to your back-end and add the `$session_id` property to your back-end events. 

Reviewer: ```suggestion
To workaround this, you can use the `posthog.onSessionId` method in the web SDK to register a callback that will be called each time the session ID changes. You can send this to your backend and add the `$session_id` property to your backend events. 
```
